<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtPaint</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/new-file.svg</normaloff>:/icons/new-file.svg</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
   <property name="toolTip">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.svg</normaloff>:/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
   <property name="toolTip">
    <string>Save image</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>New image</string>
   </property>
   <property name="toolTip">
    <string>Create image</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/paste.svg</normaloff>:/icons/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste image from clipboard</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/copy.svg</normaloff>:/icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy image to clipboard</string>
   </property>
  </action>
  <action name="actionLayers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/layers.svg</normaloff>:/icons/layers.svg</iconset>
   </property>
   <property name="text">
    <string>Layers</string>
   </property>
   <property name="toolTip">
    <string>Show layers</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/pen.svg</normaloff>:/icons/pen.svg</iconset>
   </property>
   <property name="text">
    <string>Pen Tool</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/line.svg</normaloff>:/icons/line.svg</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Line Tool</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/rectangle.svg</normaloff>:/icons/rectangle.svg</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Rectangle Tool</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/oval.svg</normaloff>:/icons/oval.svg</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="toolTip">
    <string>Ellipse Tool</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/eraser.svg</normaloff>:/icons/eraser.svg</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="toolTip">
    <string>Eraser Tool</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
